components/src/core/components/Input/Autocomplete/AutocompleteInput.vue

```diff
@@ -300,6 +300,15 @@ export default defineComponent({
       this.dropdownOpen = false;
     },
   },
+  watch: {
+    modelValue: {
+      handler(val) {
+        if (Array.isArray(this.modelValue) && this.modelValue.length === 0) {
+          this.searchTerm = null;
```

Comment:
Can this impact other widgets based on the auto complete such as multi select, single select and quick search?

Reply:
I have tested this for,

1. single select autocomplete.
2. multi select  autocomplete.
3. quick search.

and works fine